java/ql/src/semmle/code/java/frameworks/Castor.qll

```diff
@@ -0,0 +1,20 @@
+/**
+ * Provides classes and predicates for working with the Castor framework.
+ */
+
+import java
+
+/**
+ * The class `org.exolab.castor.xml.Unmarshaller`.
+ */
+class Unmarshaller extends RefType {
```

Comment:
Please rename to `CastorUnmarshaller`, otherwise, its not clear what lib this class belongs to when reading `UnsafeDeserialization`

Reply:
ok